src/test/resources/test.conf

```diff
@@ -34,6 +34,7 @@ methods {
 workspace {
   baseUrl="http://localhost:8990"
   workspacesPath="/workspaces"
+  entitiesPath="/workspaces/%s/%s/entities"
```

Comment:
Good catch - that wasn't in the last entity PR so it was on my list to add it in test.conf.